### PR TITLE
Update supported networks + fixes

### PR DESCRIPTION
**Pull request by miltonjonat**
- Using same supported networks as rollups v1.0.0
- Updating rollups dependencies to 1.0.0-rc.3 with new emulator SDK
- Removing all deployments info since we no longer have official live deployments (and previous deployments were on the Goerli network, no longer supported)
- Updating common-contracts to rollups v1.0.0
- Changed READMEs all around that mentioned Goerli, now referring to Sepolia
- Fixing testnet deployment for the template/custom-dapps

### CODE REVIEW

File: build/docker-riscv/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # syntax=docker.io/docker/dockerfile:1.4
 # layers for caching and versioning
-FROM cartesi/toolchain:0.14.0 as toolchain
-FROM cartesi/server-manager:0.7.0 as server-manager
-FROM cartesi/rollups-hardhat:0.9.1 as rollups-hardhat
+FROM cartesi/toolchain:0.15.0 as toolchain
+FROM cartesi/server-manager:0.8.1 as server-manager
```
Comment (tuler) [inline build/docker-riscv/Dockerfile:4]:
```suggestion
FROM cartesi/server-manager:0.8.2 as server-manager
```

File: build/std-rootfs/Dockerfile [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # syntax=docker.io/docker/dockerfile:1.4
 # layers for caching and versioning
-FROM cartesi/toolchain:0.14.0 as toolchain
-FROM cartesi/rootfs:0.17.0 as rootfs
-FROM cartesi/server-manager:0.7.0 as server-manager
-FROM cartesi/rollups-hardhat:0.9.1 as rollups-hardhat
+FROM cartesi/toolchain:0.15.0 as toolchain
+FROM cartesi/rootfs:0.18.0 as rootfs
+FROM cartesi/server-manager:0.8.1 as server-manager
```
Comment (tuler) [inline build/std-rootfs/Dockerfile:5]:
```suggestion
FROM cartesi/server-manager:0.8.2 as server-manager
```

File: build/docker-riscv/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+FROM cartesi/rollups-hardhat:1.0.0-rc.3 as rollups-hardhat
```
Comment (gligneul) [inline build/docker-riscv/Dockerfile:5]:
```suggestion
FROM cartesi/rollups-hardhat:1.0.0 as rollups-hardhat
```

**Review by gligneul — APPROVED** [review]

**Comment by miltonjonat** [comment]
@tuler @gligneul Added your changes (update to final 1.0.0 release + server-manager 0.8.2), can you review again?

**Comment by miltonjonat** [comment]
(btw the fixups will be squashed in the `rollups-1.0` branch)

**Review by gligneul — APPROVED** [review]

**Pull request merged**